parser/sql/dialect/oracle/src/main/antlr4/imports/oracle/DDLStatement.g4

```diff
@@ -3645,6 +3645,61 @@ permanentTablespaceClause
     )
     ;
 
+alterTablespace
+    : ALTER TABLESPACE tablespaceName
+    ( MINIMUM EXTENT sizeClause
+    | RESIZE sizeClause
+    | COALESCE
+    | SHRINK SPACE_KEYWORD (KEEP sizeClause)?
+    | RENAME TO newTablespaceName
+    | (BEGIN|END) BACKUP
+    | datafileTempfileClauses
+    | tablespaceLoggingClauses
+    | tablespaceGroupClause
+    | tablespaceStateClauses
+    | autoextendClause
+    | flashbackModeClause
+    | tablespaceRetentionClause
+    )
```

Comment:
Refer the latest oracle sql docuemnt - https://docs.oracle.com/en/database/oracle/oracle-database/21/sqlrf/ALTER-TABLESPACE.html#GUID-CA074861-55D3-4768-8995-43D4DA26365D, we should add `alter_tablespace_encryption` and `lost_write_protection` rule.